src/coreComponents/fileIO/Table/TableFormatter.hpp

```diff
@@ -55,6 +60,11 @@ class TableCSVFormatter : public TableFormatter
 {
 public:
 
+  /**
+   * @brief Construct a new Table Formatter
+   */
+  TableCSVFormatter(): TableFormatter( TableLayout()) {}
```

Comment:
I think this project code style requires:
```suggestion
  TableCSVFormatter():
    TableFormatter( TableLayout())
  {}
```